model/src/test/kotlin/PackageCurationTest.kt

```diff
@@ -168,7 +169,7 @@ class PackageCurationTest : StringSpec() {
                     ),
                     data = PackageCurationData(
                             homepageUrl = "http://home.page",
-                            vcs = VcsInfo(
+                            vcs = VcsInfoCuration(
```

Comment:
Would be nice to add a test case for emptying the VCS info of a package.

Reply:
Done in new commits.